cirq/contrib/qasm_import/qasm_test.py

```diff
@@ -11,12 +11,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import numpy as np
+
 import cirq
+import cirq.testing as ct
 
 
 # TODO as we grow the language, we'll add more complex examples here
 def test_consistency_with_qasm_output():
-    circuit1 = cirq.Circuit()
+    a, b, c, d = [cirq.NamedQubit('q_{}'.format(i)) for i in range(4)]
+    circuit1 = cirq.Circuit.from_ops(
+        cirq.Rx(np.pi / 2).on(a),
+        cirq.Ry(np.pi / 2).on(b),
+        cirq.Rz(np.pi / 2).on(b),
+        cirq.IdentityGate(1).on(c),
```

Comment:
TODO: look into qasm output for Identity

Reply:
Actually - `u3(0,0,0)` was the current output for a 1 qubit IdentityGate which is not bad, but for two qubits:

```
// Generated from Cirq v0.6.0.dev

OPENQASM 2.0;
include "qelib1.inc";


// Qubits: [q1, q2]
qreg q[2];


// Gate: I(2)
u3(pi*1.0,0,pi*0.5) q[0];
u3(0,0,0) q[1];
rx(pi*0.5) q[0];
cx q[0],q[1];
rx(pi*0.5) q[0];
ry(pi*0.5) q[1];
cx q[1],q[0];
rx(pi*-0.5) q[1];
rz(pi*0.5) q[1];
cx q[0],q[1];
u3(pi*1.0,0,pi*0.5) q[0];
u3(0,0,0) q[1];

```

Reply:
with the new commits, the two qubit version is: 

```
// Generated from Cirq v0.6.0.dev

OPENQASM 2.0;
include "qelib1.inc";


// Qubits: [0, 1]
qreg q[2];


id q[0];
id q[1];
```